products/debian12/profiles/default.profile

```diff
@@ -12,191 +12,588 @@ description: |-
     is to keep a rule in the product's XCCDF Benchmark.
 
 selections:
-    - file_permissions_backup_etc_passwd
-    - gid_passwd_group_same
-    - auditd_data_disk_full_action
-    - gnome_gdm_disable_xdmcp
-    - package_MFEhiplsm_installed
-    - configure_user_data_backups
-    - accounts_polyinstantiated_var_tmp
-    - dir_ownership_binary_dirs
+    - account_disable_post_pw_expiration
```

Comment:
Why so many new rules are added to the default profile? Also, many of them are already present in other profiles.